lib/scorecard.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class Scorecard
+  attr_reader :frames, :next_frame
+
+  def initialize(frames)
```

Comment:
maybe need a way to ensure people add only 10 frames (can add more than 10 currently)